elementary/messages/formats/block_kit.py

```diff
@@ -162,12 +183,6 @@ def _add_expandable_block(self, block: ExpandableBlock) -> None:
         Expandable blocks are not supported in Slack Block Kit.
         However, slack automatically collapses a large section block into an expandable block.
         """
-        self._add_block(
```

Comment:
adding the button label as a title in slack looked really bad, looks better without it